### PR TITLE
Issue1754

### PR DESCRIPTION
**Pull request by joernu76**

### CODE REVIEW

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
please send this to stable

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,15 @@
 Changelog
 =========
```
Comment (ReimarBauer) [inline CHANGES.rst]: there is a missing merge from stable to development. In stable we have this. see PR ;)

**Review by ReimarBauer — COMMENTED** [review]
tried it, works.

There could be a cosmetic change done. 

The other labels e.g. "Flight Tracks:" and "Open Views:" do endwith a ":". We missed that for "Operations" and "Archived Operations"
 
 
 This could also become a good first issue.

**Comment by joernu76** [comment]
Should have gone to stable. now it goes.

**Review by ReimarBauer — DISMISSED** [review]
Thx

**Comment by ReimarBauer** [comment]
after Unarchiving the bold formatted PH1 gets cleared the styles

problem

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
see comment

**Comment by ReimarBauer** [comment]
this needs a refactoring for next major too.
https://github.com/Open-MSS/MSS/blob/develop/mslib/mscolab/models.py#L136

This has to be done there because this will require a database migration too

**Comment by joernu76** [comment]
I think that bug was there before and the other pull request #1761 might fix it. At least it solves some related issues...

As you can see from the changes, this PR does not do functional changes and I'd like to keep it this way.

**Comment by joernu76** [comment]
> this needs a refactoring for next major too. https://github.com/Open-MSS/MSS/blob/develop/mslib/mscolab/models.py#L136
> 
> This has to be done there because this will require a database migration too

internally, there is a lot of renaming needed, which I mostly defer here to keep the amount of changes to stable minimal.
I will work on fixes in develop as soon as this has been merged and brought into develop.

**Comment by ReimarBauer** [comment]
This can be used for testing, on an created operation

```
>>> from mslib.mscolab.seed import archive_operation
WARNING:root:Couldn't import mscolab_auth (ImportError:'{No module named 'mscolab_auth'), creating dummy config.
>>> archive_operation("v1","aa@aa")
```

that can be later renamed for toggling the state, currently I got the same situation, I've unarchived all I had

**Comment by ReimarBauer** [comment]
ok not readed all, can remove my addition, what do you think?

**Comment by ReimarBauer** [comment]
> I think that bug was there before and the other pull request #1761 might fix it. At least it solves some related issues...
> 
> As you can see from the changes, this PR does not do functional changes and I'd like to keep it this way.

verfied that was before too

**Comment by ReimarBauer** [comment]
that seed function would be needed to add tests in stable which I also prefer for bug fixes

**Review by ReimarBauer — APPROVED** [review]
ok, does what is expected verified manually

**Pull request merged**